<ui version="4.0" >
<comment> Begin CVS Header </comment>
<comment>   $Source: /Volumes/Home/Users/shoops/cvs/copasi_dev/copasi/UI/CQTrajectoryWidget.ui,v $ </comment>
<comment>   $Revision: 1.9 $ </comment>
<comment>   $Name:  $ </comment>
<comment>   $Author: shoops $ </comment>
<comment>   $Date: 2011/09/19 13:54:23 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2011 - 2010 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

 <comment> All rights reserved. </comment>
 <class>CQTrajectoryWidget</class>
 <widget class="TaskWidget" name="CQTrajectoryWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>216</height>
   </rect>
  </property>
  <property name="autoFillBackground" >
   <bool>false</bool>
  </property>
  <property name="caption" stdset="0" >
   <string>Time Course</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="mpGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="mpLblDuration" >
       <property name="text" >
        <string>Duration</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="mpEditDuration" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="mpLblIntervalSize" >
       <property name="text" >
        <string>Interval Size</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="mpEditIntervalSize" />
     </item>
     <item row="1" column="2" >
      <widget class="QLabel" name="mpLblIntervals" >
       <property name="text" >
        <string>Intervals</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3" >
      <widget class="QLineEdit" name="mpEditIntervals" />
     </item>
     <item row="2" column="1" colspan="3" >
      <layout class="QHBoxLayout" name="mpLayoutDelayed" >
       <item>
        <widget class="QCheckBox" name="mpCheckDelay" >
         <property name="text" >
          <string>Suppress Output Before</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="mpEditDelay" />
       </item>
       <item>
        <spacer name="mpSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="1" colspan="3" >
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QCheckBox" name="mpCheckOutputEvent" >
         <property name="text" >
          <string>Output Events  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mpCheckSave" >
         <property name="text" >
          <string>Save Result in Memory</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="0" colspan="4" >
      <widget class="Line" name="mpLine2" >
       <property name="frameShape" >
        <enum>QFrame::HLine</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0" >
      <widget class="QLabel" name="mpLblIntegrationInterval" >
       <property name="text" >
        <string>Integration Interval</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1" >
      <widget class="QLineEdit" name="mpEditIntegrationInterval" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="2" >
      <widget class="QLabel" name="mpLblOutputInterval" >
       <property name="text" >
        <string>Output Interval</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="3" >
      <widget class="QLineEdit" name="mpEditOutputInterval" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="4" >
      <widget class="Line" name="mpLine1" >
       <property name="frameShape" >
        <enum>QFrame::HLine</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>TaskWidget</class>
   <extends>QWidget</extends>
   <header>TaskWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mpEditDuration</tabstop>
  <tabstop>mpEditIntervalSize</tabstop>
  <tabstop>mpEditIntervals</tabstop>
  <tabstop>mpEditDelay</tabstop>
  <tabstop>mpCheckSave</tabstop>
 </tabstops>
 <includes>
  <include location="local" >TaskWidget.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>mpEditDuration</sender>
   <signal>editingFinished()</signal>
   <receiver>CQTrajectoryWidget</receiver>
   <slot>slotDuration()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpEditIntervalSize</sender>
   <signal>editingFinished()</signal>
   <receiver>CQTrajectoryWidget</receiver>
   <slot>slotIntervalSize()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpEditIntervals</sender>
   <signal>editingFinished()</signal>
   <receiver>CQTrajectoryWidget</receiver>
   <slot>slotIntervals()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpEditDelay</sender>
   <signal>editingFinished()</signal>
   <receiver>CQTrajectoryWidget</receiver>
   <slot>updateIntervals()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpCheckDelay</sender>
   <signal>toggled(bool)</signal>
   <receiver>CQTrajectoryWidget</receiver>
   <slot>slotOutputDelay(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
